.github/workflows/main.yml

```diff
@@ -42,10 +44,24 @@ jobs:
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
+      - name: Cleanup build folder
+        run: |
+          ls -la ./
+          rm -rf ./* || true
+          rm -rf ./.??* || true
+          ls -la ./
```

Comment:
Hi @efeyazgan why the powheg part is also modified

Reply:
Hi, because Markus discovered that the space is filling up with these workflows (which was blocking the workflows). So, we added this cleaning both to powheg and request checking script parts.

Reply:
@menglu21 @bbilin Any issues remianing for this PR? thanks.